src/waveform/renderers/waveformmark.cpp

```diff
@@ -5,21 +5,25 @@
 
 #include "waveformmark.h"
 
-WaveformMark::WaveformMark(int hotCue)
+WaveformMark::WaveformMark( const QString& group,
+                            const QDomNode& node,
+                            const SkinContext& context,
+                            const WaveformSignalColors& signalColors,
+                            int hotCue)
     : m_iHotCue(hotCue) {
-}
-
-void WaveformMark::reset(int hotCue) {
-    WaveformMark(hotCue).swap(*this);
-}
-
-void WaveformMark::setup(const QString& group, const QDomNode& node,
-                         const SkinContext& context,
-                         const WaveformSignalColors& signalColors) {
+    m_pPointCos = nullptr;
```

Comment:
This should not make a difference, since a uniqu_ptr is empty by default:
http://en.cppreference.com/w/cpp/memory/unique_ptr/unique_ptr